ts/Dashboards/Components/HTMLComponent.ts

```diff
@@ -85,7 +95,7 @@ class HTMLComponent extends Component<HTMLComponent.HTMLComponentEvents> {
 
         const component = new HTMLComponent(
             merge(
-                options,
+                options as any,
```

Comment:
Can we avoid that?

Reply:
For now, there is no way of avoiding that. IMO, it can be changed later, when we tackle the JSON options and its interfaces.